gapis/api/transform/transform.go

```diff
@@ -33,7 +33,7 @@ type StateMutator func(cmds []api.Cmd) error
 // they aim to do and emit a list of commands to pass it to the next transform.
 type Transform interface {
 	// BeginTransform is called before transforming any command.
-	BeginTransform(ctx context.Context, inputCommands []api.Cmd, inputState *api.GlobalState) ([]api.Cmd, error)
+	BeginTransform(ctx context.Context, inputState *api.GlobalState) error
```

Comment:
We've agreed with Pascal to keep this change. But for the record on this code review: I still think this function should have a symmetrical signature with EndTransform() so that commands can be inserted at the start of a replay as well as the end.